README.md

```diff
@@ -46,11 +46,12 @@ This will install the initializer:
 config/initializers/client_side_validations.rb
 ```
 
-If you want to copy the asset files from the gem into your project:
+Assets will automatically be loaded. However if you want to copy the asset files from the gem into your project:
```

Comment:
this is not true, you need to add 

```js
//= require rails.validations
```

to application.js, but I've already merge this PR.